docs/tools/cmor.rst

```diff
@@ -75,7 +75,7 @@ background
 ~~~~~~~~~~
 
 The bulk of this routine is housed in ``fre/cmor/cmor_mixer.py``, which is a rewritten version of
-Sergey Malyshev's original ``CMORcommander.py`` script, utilized during GFDL's CMIP6 publishing run.
+Sergey Nikonov's original ``CMORcommander.py`` script, utilized during GFDL's CMIP6 publishing run.
```

Comment:
CMORcommander was a prototype rewrite of fremetar, which GFDL used for CMIP6 publishing.